<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="llaves">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>290</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="num_array" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>651</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black</string>
   </property>
   <property name="verificar" stdset="0">
    <bool>true</bool>
   </property>
   <widget class="QTableWidget" name="senal_mat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="grafica_canales">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color: rgb(169, 228, 255); </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="graficar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_canal">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
border: 2px solid black;
background-color: rgba(255, 255, 255,0.8);   </string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CANAL:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="canales">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>390</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="volver">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>560</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cerrar">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>600</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>CERRAR SESIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt bold &quot;Malgun Gothic&quot;; 
background-color: rgba(255, 255, 255,0.5);   
border: 2px solid rgba(0,0,0,0.7); </string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Tener en cuenta la siguiente información&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="todos">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>390</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>Todos los canales</string>
   </property>
  </widget>
  <widget class="QPushButton" name="canal">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>Un canal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>270</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Elegir información a graficar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retroceder_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>Retroceder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adelantar_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>Adelantar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retroceder_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; 
</string>
   </property>
   <property name="text">
    <string>Retroceder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adelantar_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>Adelantar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>-170</y>
     <width>1121</width>
     <height>951</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>foto analisis.jpeg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;

</string>
   </property>
   <property name="text">
    <string>Filas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filas">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
color font:black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>110</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>Columnas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="columnas">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
color font: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>Mínima:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_min">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>Máxima:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_max">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CLICK ENTER&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: 2px solid black;
background-color: rgba(255, 255, 255,0.8);   </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>90</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: 2px solid black;
background-color: rgba(255, 255, 255,0.8);   </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt bold &quot;Malgun Gothic&quot;; 
background-color: rgba(255, 255, 255,0.5);   
border: 2px solid rgba(0,0,0,0.7); </string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Configuración de gráficas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color: rgba(255, 255, 255,0.8);   </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color: rgba(255, 255, 255,0.8);   </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>270</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RECOTE DE SEÑALES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>num_array</zorder>
  <zorder>graficar</zorder>
  <zorder>num_canal</zorder>
  <zorder>canales</zorder>
  <zorder>volver</zorder>
  <zorder>cerrar</zorder>
  <zorder>label_3</zorder>
  <zorder>todos</zorder>
  <zorder>canal</zorder>
  <zorder>retroceder_1</zorder>
  <zorder>adelantar_1</zorder>
  <zorder>retroceder_2</zorder>
  <zorder>adelantar_2</zorder>
  <zorder>label_9</zorder>
  <zorder>label_4</zorder>
  <zorder>filas</zorder>
  <zorder>label_10</zorder>
  <zorder>label_5</zorder>
  <zorder>columnas</zorder>
  <zorder>label_11</zorder>
  <zorder>grafica_canales</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>p_min</zorder>
  <zorder>p_max</zorder>
  <zorder>label_8</zorder>
  <zorder>label_6</zorder>
  <zorder>llaves</zorder>
  <zorder>label_14</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
